integrations/dd-sdk-android-okhttp/src/test/kotlin/com/datadog/android/okhttp/trace/TracingInterceptorNotSendingSpanTest.kt

```diff
@@ -1324,19 +1325,23 @@ internal open class TracingInterceptorNotSendingSpanTest {
         return "$protocol://${host.lowercase(Locale.US)}/$path"
     }
 
-    private fun forgeRequest(
+    protected fun forgeRequest(
         forge: Forge,
         configure: (Request.Builder) -> Unit = {}
     ): Request {
         val builder = Request.Builder().url(fakeUrl)
         if (forge.aBool()) {
-            fakeMethod = "POST"
+            fakeMethod = RumHttpMethod.POST
```

Comment:
> [!IMPORTANT]
Same as above, we might want to fake all possible kinds of requests, so maybe have either `POST`, `PATCH` or `PUT` here